Source: https://bugs.launchpad.net/bzr/+bug/461149
Upstream: See Source
Reason: Fix test failure when paramiko isn't installed

=== modified file 'bzrlib/tests/test_selftest.py'
--- bzrlib/tests/test_selftest.py	2009-08-25 03:48:15 +0000
+++ bzrlib/tests/test_selftest.py	2009-08-25 06:55:08 +0000
@@ -1922,8 +1922,8 @@
     def test_transport_sftp(self):
         try:
             import bzrlib.transport.sftp
-        except ParamikoNotPresent:
-            raise TestSkipped("Paramiko not present")
+        except errors.ParamikoNotPresent:
+            raise tests.TestSkipped("Paramiko not present")
         self.check_transport_set(bzrlib.transport.sftp.SFTPAbsoluteServer)
 

